content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -368,41 +368,22 @@ Here's a breakdown of what/why:
    certificates from the `kube-apiserver` when the certificate expiration approaches.
 * `--cert-dir`the directory where the TLS certs are located.
 
-### Use kubeadm with other CRI runtimes
+### Use kubeadm with containerd
 
-Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
-The container runtime used by default is Docker, which is enabled through the built-in
-`dockershim` CRI implementation inside of the `kubelet`.
+From v1.12.0 the suggested kubeadm CRI is containerd. For further information refer to [CRI Installation](/docs/setup/cri/cri-installation/) instructions.
```

Comment:
Can you elaborate on this a bit? Why containerd is suggested kubeadm CRI? Why not CRI-O or both?

Reply:
I believe this was due to the lack of tests with docker, not totally sure about CRI-O. I'll defer to @timothysc.

Reply:
@neolit123 once we have CI-Signal back we need to double check the versions.